src/main/java/seedu/duke/utils/State.java

```diff
@@ -1,5 +1,19 @@
 package seedu.duke.utils;
 
+import seedu.duke.model.SelectedModule;
+
+import java.util.List;
+
 public class State {
 
+    public int semester = 0;
```

Comment:
I think this should be private with getter and setter

Reply:
updated, thanks